modules/gazelle/metadata.json

```diff
@@ -36,7 +36,8 @@
         "0.37.0",
         "0.38.0",
         "0.39.0",
-        "0.39.1"
+        "0.39.1",
```

Comment:
@meteorcloudy Should I update the `repository` field to point to `bazel-contrib` even though that's not correct for previous versions?

Reply:
you could have multiple entries there to be safe